### ISSUE TITLE
MacOS Version / Linux

### ISSUE THREAD

**Issue by tresloukadu** [notepad-plus-plus/notepad-plus-plus#7548]
Man sorry for posting in the wrong topic. However I would like to stress how I miss notepad++ in my MacOS. I like it not for development purposes since we have a plenty of powerful IDE's. But I like Notepad++ to edit process and manipulate raw text, it is easy and useful for tasks like add or remove UTF-8 BOM (byte-order-mark), regex, etc. It has tools to assist you manipulate text and as far as I know I didn't find such tool for Mac. When I try to find some equivalent people tell me to use Aton, Sublime but this is not the case. When you guys will develop a cross platform version? Or at least a MacOS native one? Do you have a side project of notepad++ for mac?

**Comment by Halamix2** [comment]
tl;dr: NPP is tightly connected with Win32 API
#108
#332
#6151
